### PR TITLE
Add UT Test of TestGetVolumeName to cephfs_test.go

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
Add UT Test of TestGetVolumeName to cephfs_test.go
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/volume/cephfs/cephfs_test.go [hunk 1]
```diff
@@ -245,3 +245,89 @@ func TestGetAccessModes(t *testing.T) {
...
+	plugMgr.InitPlugins(ProbeVolumePlugins(), nil /* prober */, volumetest.NewFakeVolumeHost(tmpDir, nil, nil))
+
+	plug, err := plugMgr.FindPersistentPluginByName("kubernetes.io/cephfs")
+	if err != nil {
+		t.Errorf("Can't find the plugin by name")
+	}
+	tests := []testcase{
+		{
+			name:      "persistent volume source",
+			defaultNs: "default",
```
Comment (cofyc) [inline pkg/volume/cephfs/cephfs_test.go]: `defaultNs` not used

File: pkg/volume/cephfs/cephfs_test.go [hunk 2]
```diff
@@ -245,3 +245,89 @@ func TestGetAccessModes(t *testing.T) {
...
+							SecretFile: "/etc/ceph/user.secret",
+						},
+					},
+				},
+			},
+			expectedError: nil,
+		},
+	}
+	for _, testcase := range tests {
+		_, err := plug.GetVolumeName(testcase.spec)
```
Comment (cofyc) [inline pkg/volume/cephfs/cephfs_test.go]: `GetVolumeName` calls `getVolumeSource` to get volume name without any extra logic.
How about testing `getVolumeSource` directly and check its other returned values too?

File: pkg/volume/cephfs/cephfs_test.go (hunk 1 above)
Comment (jianglingxia) [reply pkg/volume/cephfs/cephfs_test.go]: sorry,miss it! fixed it!

File: pkg/volume/cephfs/cephfs_test.go (hunk 2 above)
Comment (jianglingxia) [reply pkg/volume/cephfs/cephfs_test.go]: thanks for your review and your advice !yeah, GetVolumeName calls getVolumeSource ;
but if directly testing getVolumeSource ,the GetVolumeName does not test,also the GetVolumeName return many values,user no need care the value but err I think maybe I wrong

**Comment by jianglingxia** [comment]
ping @jsafrane 
really thanks!

**Pull request closed without merging**